application/app/services/repo/base_resolver.rb

```diff
@@ -0,0 +1,17 @@
+module Repo
```

Comment:
should we use an informal module name `Repo` or a more formal like `Repository` ?

Reply:
I named initially Repository, but it felt too long and redundant. Repo felt more natural and widely understood